src/metrics/iroh.rs

```diff
@@ -0,0 +1,16 @@
+use std::fmt;
+
+use prometheus_client::{metrics::counter::Counter, registry::Registry};
+use tracing::error;
+
+use crate::{
+    metrics::core::Collector,
+    metrics::core::{HistogramType, MRecorder, MetricType, MetricsRecorder},
+};
+
+make_metrics! {
+    Iroh,
+    RequestsTotal: Counter: "Total number of requests received",
+    BytesSent: Counter: "Number of bytes streamed",
+    BytesReceived: Counter: "Number of bytes received"
```

Comment:
are these metrics actually recorded?

Reply:
They are not used, mostly set as an example and I've used them for testing stuff out. I expect to refactor these as soon as we start collecting real metrics.